MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -1183,8 +1182,8 @@ private void addStar(ArrayList<Entity> list, String faction, int skill, int qual
                 list.add(en);
             }
         }
-        if (forceType == FORCE_NOVA || forceType == FORCE_VEENOVA) {
-            unitType = forceType == FORCE_VEENOVA ? UnitType.INFANTRY : UnitType.BATTLE_ARMOR;
+        if (forceType == FORCE_NOVA) {
```

Comment:
I don't see a reason to eliminate the VEENOVA logic.

Reply:
Unused and has never been used, years after it was originally implemented.